File: sorting/selectionsort/SelectionSort.java
import java.util.Arrays;
public class SelectionSort
{
    public static void  main(String args[])
    {
        int arr[]=new int[]{64,25,12,22,11};
        System.out.println("Unsorted : "+Arrays.toString(arr));

        for(int i=0;i<arr.length-1;i++)
        {
            int minIndex=i;
            for(int j=i+1;j<arr.length;j++)
            {
                if(arr[minIndex]>=arr[j])
                {
                    minIndex=j;
                }
            }
            if(minIndex!=i)
            {
                int temp=arr[i];
                arr[i]=arr[minIndex];
                arr[minIndex]=temp;
            }
        }
        System.out.println("Sorted : "+Arrays.toString(arr));
    }
}

File: sorting/selectionsort/selectionsort.js
function selectionSort(arr){
    let n=arr.length;
    for(let i=0;i<n-1;i++)
        {
            let minindex=i;
            for(let j=i+1;j<n;j++)
            {
                if(arr[minindex]>=arr[j])
                {
                    minindex=j;
                }
            }

            if(i!=minindex)
            {
                [arr[i], arr[minindex]] = [arr[minindex], arr[i]];
            }
        }
}
let arr1=[64,25,12,22,11]
console.log(`unsorted : ${arr1}`)
selectionSort(arr1);
console.log(`sorted : ${arr1}`)
console.log(`length :${arr1.length}`)